wrapped.go

```diff
@@ -0,0 +1,61 @@
+package rules
+
+type WrappedRuleset struct {
+	StandardRuleset
+}
+
+func replace(value, min, max int32) int32 {
+	if value < min {
+		return max
+	}
+	if value > max {
+		return min
+	}
+	return value
+}
+
+func (r *WrappedRuleset) CreateNextBoardState(prevState *BoardState, moves []SnakeMove) (*BoardState, error) {
+	nextState := prevState.Copy()
+
+	err := r.moveSnakes(nextState, moves)
+	if err != nil {
+		return nil, err
+	}
+
+	err = r.reduceSnakeHealth(nextState)
+	if err != nil {
+		return nil, err
+	}
+
```

Comment:
A `maybeDamageHazards` step was added here in `StandardRuleset` that we probably want to reproduce here.

Reply:
added